packages/legacy/core/App/modules/openid/screens/OpenIDCredentialOffer.tsx

```diff
@@ -27,7 +27,7 @@ type OpenIDCredentialDetailsProps = StackScreenProps<DeliveryStackParams, Screen
 
 const OpenIDCredentialDetails: React.FC<OpenIDCredentialDetailsProps> = ({ navigation, route }) => {
 
-
+    // FIXME: change params to accept credential id to avoid 'non-serializable' warnings
```

Comment:
I had a conversation with @bryce-mcmath and decided to not fix this as this feature is still being worked on